roles/deploy/tasks/deploy.yml

```diff
@@ -73,6 +73,16 @@
     - skip_ansible_lint
   notify: restart unicorn
 
+# Install JS dependencies with yarn
```

Comment:
why not naming the task below exactly this way? then there's no need for the comment

Reply:
I'd always do that! But here I am keeping consistency, the file is organized in groups with comments and if I remove the comment this task would "appear" as part of the previous group.
This is also done below:
![image](https://user-images.githubusercontent.com/1640378/83257928-41963a80-a1ad-11ea-9118-7535b30f749d.png)

This is easier to see if you open the file in your IDE and scroll.